lib/read.js

```diff
@@ -23,6 +23,16 @@ var zlib = require('zlib')
 
 module.exports = read
 
+/**
+ * @const
+ * whether current node version has brotli support
+ */
+var hasBrotliSupport = false
+try {
+  hasBrotliSupport = 'brotli' in require('process').versions
```

Comment:
The process module is just a global so no need to require it or have in a try catch.